snaps_openstack/ansible_p/ansible_utils/variable.yaml

```diff
@@ -28,11 +24,15 @@ DOCKER_SERVICE_SRC: "/lib/systemd/system/docker.service"
 DOCKER_SERVICE_DEST: "/etc/systemd/system/docker.service"
 DAEMON_DOCKER_FILE_DEST: "/etc/docker/daemon.json"
 LOCAL_SETTING_HORIZON: "/etc/kolla/horizon/local_settings"
-GIT_CLONE_PATH: "/etc/OpenStack/kolla"
+GIT_CLONE_PATH_QUEENS: "/etc/OpenStack/kolla-ansible"
```

Comment:
You should probably rename this back to 'GIT_CLONE_PATH' as this codebase will only be supporting one OpenStack release.